schemas/learningResource.schema.tpl.json

```diff
@@ -0,0 +1,49 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LearningResource",
+  "_extends": "creativeWork.schema.tpl.json",
+  "required": [
+    "name",
+    "about",
+    "datePublished"
+  ],
+  "properties": {
+    "competencyRequired": {
+        "type": "string",
+        "_instruction": "Enter the knowledge, skills, or abilities that are prerequisites for being able to use the learning resource."
+    },
+    "educationalLevel": {
+        "type": "string",
+        "_instruction": "Enter the level of the resource in terms of progression through an educational or training context, for example 'beginner', 'intermediate' or 'advanced'"
+    },
+    "learningResourceType": {
+      "_instruction": "Add the type of the learning resource",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/LearningResourceType",
+      ]
+    },
+    "teaches": {
```

Comment:
should this not be already captured in the abstract of the creative work (or differently: what should be entered in the abstract for a learning resource compared to this one here)?

Reply:
I was thinking the abstract would be more general/descriptive, whereas this field would be more of a list of bullet points.

Reply:
okay